huggingface/pytorch/training/docker/1.11/py3/cu113/Dockerfile.trcomp.gpu

```diff
@@ -0,0 +1,109 @@
+# https://github.com/aws/deep-learning-containers/blob/master/available_images.md 
+# refer to the above page to pull latest Pytorch image
+
+# docker image region us-west-2
+FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-training:1.11.0-gpu-py38-cu113-ubuntu20.04-sagemaker
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+# version args
+ARG TRANSFORMERS_VERSION=4.19.2
+ARG DATASETS_VERSION=1.18.4
+ARG PYTHON=python3
+
+ARG PT_BUCKET=https://sm-training-comp-pytorch-binaries.s3.us-west-2.amazonaws.com/617bcafe-7da3-4700-a4ba-6707a11c6630/20220526-213653/9ad1460a3411c47cc696c97b52d11dd3984fd2b1
```

Comment:
[PT 1.11 Base DLC ](https://github.com/aws/deep-learning-containers/blob/master/pytorch/training/docker/1.11/py3/cu115/Dockerfile.gpu#L11) shipped with [TorchData](https://pytorch.org/data/beta/index.html). Please check if we need to rebuild this with trcomp support.